02-express-tutorial/public/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello World</h1>
+    <div id="data-div">
+
+    </div>
+    <button>click here!</button>
```

Comment:
You need to add the onclick event and call the handleClick function here.
Example: 
`<button onclick="handleClick()">click here!</button>`